js/src/forum/components/Post.js

```diff
@@ -71,8 +72,8 @@ export default class Post extends Component {
               )}
             </ul>
           </aside>
-          <footer className="Post-footer">
-            <ul>{listItems(this.footerItems().toArray())}</ul>
+          <footer className={classList('Post-footer', !footerItems.length && 'Post-footer--empty')}>
+            <ul>{listItems(footerItems)}</ul>
```

Comment:
How about
```jsx
<footer className="Post-footer">
  {footerItems.length && (
    <ul>{listItems(footerItems)}</ul>
  ) || ''}
```

and then use the fancy `:empty` in CSS, so 
```CSS
&:empty {
  display: none;
}
```

:P too much nitpicking ? sorry..

Reply:
Oh I didn't think about that! That's a better idea! :P

Not sure how it'll handle a blank text node, if Mithril inserts one, as the empty pseudoselector doesn't match elements with any content (whitespace, even).

Reply:
>Not sure how it'll handle a blank text node, if Mithril inserts one, as the empty pseudoselector doesn't match elements with any content (whitespace, even).

true 🤔